### ISSUE TITLE
loss nan problem

### ISSUE THREAD

**Issue by manutdzou** [hunglc007/tensorflow-yolov4-tflite#197]
this may be mish bugs, even I clip the gradient value and descend the lr to even 1e-6.
I also load the darknet weights and can only get about 50mAP@0.5 .Then I pretrain by this darknet weight also get nan quickly.

**Comment by duanfxld** [comment]
did u solve the problem? Can u share some experience?
